draft-ietf-tls-tls13.md

```diff
@@ -1517,7 +1517,7 @@ Structure of this message:
            Random random;
            opaque legacy_session_id<0..32>;
            CipherSuite cipher_suites<2..2^16-2>;
-           opaque legacy_compression_methods<1..2^8-1>;
+           opaque legacy_compression_methods[2] = 0x0100;
```

Comment:
Just to clarify: we cannot do this for legacy_session_id because of the stated case, but it is okay for legacy_compression_methods?  (If you send a 1.3 CH, we say these bytes MUST be exactly as listed.)